cmd/tendermint/commands/root.go

```diff
@@ -29,12 +29,20 @@ func registerFlagsRootCmd(cmd *cobra.Command) {
 
 // ParseConfig retrieves the default environment configuration,
 // sets up the Tendermint root and ensures that the root exists
-func ParseConfig() (*cfg.Config, error) {
+func ParseConfig(cmd *cobra.Command) (*cfg.Config, error) {
```

Comment:
It may be better to pass the home path string instead of an entire command but we can clean this up in a later PR

Reply:
thx for your comment. I'll clean this up on next issue.